sql/002_create_smallcompanies.sql

```diff
@@ -0,0 +1,11 @@
+CREATE TABLE `companies` (
+  `c_id` int NOT NULL,
+  `c_name` varchar(45) NOT NULL,
+  `services_offered` varchar(50) NOT NULL,
+  `city` varchar(45) NOT NULL,
+  `email` varchar(45) NOT NULL,
+  `company_phone` varchar(45) NOT NULL,
+  `number_of_volunteers` int NOT NULL,
+  `perks` varchar(45) NOT NULL,
```

Comment:
Perks has much more data - it won't fit into 45 characters.

I suggest creating a separate table for perks, with a foreign key that references companies.

Reply:
Also, please see the wireframes and ask @lisanag, @Kenthia11 and @ankoanju about the data they want to see in the frontend for perks.